src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/nomis/UserApiService.kt

```diff
@@ -82,10 +82,11 @@ class UserApiService(
     return serviceWebClientUtils.getIgnoreError("/users/${username.uppercase()}", PrisonUser::class.java)
   }
 
+  @Throws(EntityNotFoundException::class)
```

Comment:
We don't need the @throws in kotlin

Reply:
I have created an integration test to cover that as suggested. Thanks